sql_metadb/derived_tables/holding_notes.sql

```diff
@@ -0,0 +1,45 @@
+DROP TABLE IF EXISTS holdings_notes;
+CREATE TABLE IF EXISTS holdings_notes;
+WITH notes AS (
+    SELECT 
+    h.instanceid AS instance_id,
+    h.id AS holdings_id,
+    jsonb_extract_path_text(h.jsonb, 'hrid') AS holdings_hrid,
+    jsonb_extract_path_text(notes.jsonb, 'note') AS note,
+    jsonb_extract_path_text(notes.jsonb, 'holdingsNoteTypeId')::uuid AS holdings_note_type_id,

```

Comment:
You have the note type id. You also need to include the note type name. I didn't see that.